.github/workflows/tests.yml

```diff
@@ -45,8 +45,17 @@ jobs:
           python-version: "3.11"
       - run: ./tests/check_new_syntax.py
 
+  ruff:
+    name: Lint with Ruff
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: chartboost/ruff-action@v1
+        with:
+          version: "0.0.292" # must match .pre-commit-config.yaml and requirements-test.txt
```

Comment:
Well, the version doesn't match. :)

Reply:
Thanks, I got distracted and forgot to check for other locations after merging